plugins/_general/config.schema.json

```diff
@@ -1,37 +1,73 @@
 {
-    "schema": {
-        "general": {
-            "type": "object",
-            "title": "General Settings",
-            "properties": {
-                "language": {
-                    "default": "en-US",
-                    "type": "string",
-                    "title": "Language",
-                    "required": true
-                },
-                "layout": {
-                    "default": "main",
-                    "type": "string",
-                    "title": "Layout",
-                    "required": true
-                }
-            }
-        }
-    },
-    "form":[
-      {
-        "key":"general",
-        "type":"fieldset",
-        "title":"General Settings",
-        "expandable":false,
-        "order":-100
-      }
-    ],
-    "value":{
-        "general":{
-            "language": "en-US",
-            "layout":"main"
-        }
-    }
-}
+	"schema": {
```

Comment:
@justbill2020 The config.schema.json is 4 spaces instead of tabs. We should make a PR just to change all of this.  This way Sam's changes will not show the whole file changed.